wcfsetup/install/files/js/WoltLabSuite/Core/Ui/File/Delete.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * Delete files which are uploaded via AJAX.
+ *
+ * @author	Joshua Ruesweg
+ * @copyright	2001-2018 WoltLab GmbH
```

Comment:
```suggestion
 * @copyright	2001-2019 WoltLab GmbH
```

(in the other files, too)